examples-next/basic/admin/fieldViews/Content.tsx

```diff
@@ -138,36 +154,100 @@ export const componentBlocks = {
       authors: fields.relationship<'many'>({ label: 'Authors', relationship: 'featuredAuthors' }),
     },
   }),
-  panel: component({
-    component: ({ content }) => {
-      return <div>{content}</div>;
+  notice: component({
+    component: ({ content, intent }) => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
I'd image the rule would be happy if you did `function Notice`(still inline though)

Reply:
Could you add that using the `suggestion` syntax? Not quite sure what you mean